thetanuts/README.md

```diff
@@ -5,11 +5,12 @@ Python SDK to interact with Thetanuts Finance auctions on Paradigm
 ## Setup to run sample thetanuts test script
 
 Open a container with python
-$ docker run -it --rm \
-    -v $(pwd):/tmp/code -w /tmp/code \
-    -p 8888:8888 \
-    python:3.10 \
-    bash
+
+    $ docker run -it --rm \
```

Comment:
Why this change?

Reply:
Wasn't formatted as code, indented to format properly.
Unrelated with the main topic of this PR, but it's a so small change that does not worth a dedicated PR to fix it

Reply:
Ok, just checking if it was intended.